plugins/CuraEngineBackend/StartSliceJob.py

```diff
@@ -102,6 +118,9 @@ def get_value(self, expression: str, args: [str], kwargs: dict) -> str:
         setting_function = SettingFunction(expression)
         value = setting_function(container_stack, additional_variables=additional_variables)
 
+        print("value", value)
+        print("expression", expression)
```

Comment:
```suggestion
```

Looks like debug prints to me, so we should probably remove em.

Reply:
oops